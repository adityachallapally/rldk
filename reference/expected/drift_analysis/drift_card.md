# Drift Detection Card

## ✅ No Drift Detected

The runs appear to be consistent within the specified tolerance.

## 📁 Report Location

Full report saved to: `reference/expected/drift_analysis/drift_card.md`
Detailed events saved to: `reference/expected/drift_analysis/diff_events.csv`

## 🔍 Analysis Parameters

- **Total divergence events:** 0
