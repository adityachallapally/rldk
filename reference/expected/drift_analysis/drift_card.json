{
  "version": "1",
  "diverged": false,
  "first_step": null,
  "signals_monitored": [
    "sample_id",
    "input_ids_sha256",
    "attention_mask_sha256",
    "outputs.text",
    "reward_scalar",
    "loss"
  ],
  "k_consecutive": 3,
  "window_size": 50,
  "tolerance": 2.0
}